_config.yml

```diff
@@ -14,6 +14,7 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 title: Hamburg English Speaking Catholic Community
+short_title: HESCC
```

Comment:
I'm not sure the acronym "HESCC" means much (or should, really). So maybe just "Hamburg Catholics" (to match the domain) instead?

Reply:
You're tight, Hamburg Catholics sounds much better